lib/rules/property-units.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var helpers = require('../helpers');
+
+module.exports = {
+  'name': 'property-units',
+  'defaults': {
+    'unitsAllowedForProperties': {},
+    'globallyAllowedUnits': []
+  },
+  'detect': function (ast, parser) {
+    var result = [];
+    var globallyAllowedUnits = parser.options.globallyAllowedUnits;
+    var unitsAllowedForProperties = parser.options.unitsAllowedForProperties;
+
+    ast.traverseByType('declaration', function (declaration) {
+      var declarationType = declaration.content[0].content[0].type,
+          propertyName = declaration.content[0].content[0].content,
+          valueNode = declaration.content
+            .filter(function (node) {
+              return node.type === 'value';
+            })[0];
+      if (declarationType === 'ident' && valueNode.content[0].type === 'dimension') {
+        var dimensionNode = valueNode.content[0];
+        var unitType = dimensionNode.content[1].content;
```

Comment:
I don't think this is something we've been enforcing but within rules it makes sense to stick with one style.